Tests/CryptomatorCloudAccessIntegrationTests/Box/BoxCredentialMock.swift

```diff
@@ -8,22 +8,48 @@
 
 import Foundation
 import Promises
-@testable import BoxSDK
+@testable import BoxSdkGen
 #if canImport(CryptomatorCloudAccessCore)
 @testable import CryptomatorCloudAccessCore
 #else
 @testable import CryptomatorCloudAccess
 #endif
 
+// Custom in-memory token storage
+class InMemoryTokenStore: TokenStorage {
```

Comment:
Is this really needed? Can't this be used? https://github.com/box/box-swift-sdk-gen/blob/62670ce06100cbabae3482b3a89c2365b6322d57/Sources/Box/InMemoryTokenStorage.swift

Reply:
Removed in commit d544b20.